Coffee3.4/src/CoffeeBrush.java

```diff
@@ -0,0 +1,206 @@
+import java.util.Scanner; //import Scanner from library
+public class CoffeeBrush {
+
+    private enum State {
+        Ready, SelectCoffee, Off, Filling
+    }
+
+    public Scanner scan = new Scanner(System.in);
+
+    /**
+     * buy options
+     * */
+    private final int espresso = 1;
+    private final int latte = 2;
+    private final int cappuccino = 3;
+
+    /**
+     * initial resources
+     * */
+    private int water = 400;
+    private int milk = 540;
+    private int coffee = 120;
+    private int cups = 9;
+    private int money = 550;
+
+    private State state = State.Ready;
+
+    CoffeeBrush(){
+        setReady();
+    }
+
+    /**
+     * one cup of espresso contains:
+     * */
+    private int e_water = 250;
```

Comment:
@aniterstepanyan always define fields in top of the class before other methods or constructor

Reply:
Done